example/apollo-server/movies.js

```diff
@@ -1,6 +1,6 @@
 import { augmentTypeDefs, augmentSchema } from '../../src/index';
 import { ApolloServer, gql, makeExecutableSchema } from 'apollo-server';
-import { v1 as neo4j } from 'neo4j-driver';
+import neo4j from 'neo4j-driver';
```

Comment:
I'm not sure if this should be in this pull request, but the example server didn't work for on the master branch.

Reply:
This is a result of upgrading the version of neo4j-driver, which removes the v1 namespace. I was lazy and only updated movies-middleware.js which is why this one was failing. My fault.